README.md

```diff
@@ -47,13 +47,17 @@ After you have installed the CLI, you can run the following from the root of the
 ```bash
 diesel migration run --database-url <DATABASE_URL>
 ```
+Where ```<DATABASE_URL>``` uses the ```postgess://``` URI Example:
+```postgres://username:password@localhost:5432/database_name```
 
 You can then run:
 
 ```bash
 cargo shuttle run
 ```
 
+NOTE: If ```cargo shuttle run``` fails one must first run the command ```cargo install cargo-shuttle```
```

Comment:
We should just add this as a step above running cargo shuttle run instead. We can add it to the same command or we can add an entire small section above the `cargo shuttle run` step.

For example, we could change it to:
```bash
cargo install cargo-shuttle && cargo shuttle run
```

Otherwise we could add verbiage above saying:
As this bot deploys on Shuttle, it uses Shuttle secret store and Shuttle async runtime. As such, you need the `cargo-shuttle` crate installed on your local machine to run the bot. You can install this by running:
```bash
cargo install cargo-shuttle
```

Reply:
fixed